### ISSUE TITLE
How to check the running php version of a specific site

### ISSUE THREAD

**Issue by Corvalan** [omega8cc/boa#1346]
I tried the php.info file method in the root folder of a site,
`<?php phpinfo(); ?>`
but i get page not found error

Another method maybe?

**Comment by ar-jan** [comment]
You can just look at the status report inside Drupal, this lists the version and phpinfo results.

**Comment by Corvalan** [comment]
Yes I know, but the website in my case is not accessible because of errors and it probably is because of my recent upgrade of the BOA platform to php 7.3. I now have assigned the website to its original php version 5.6 in ~/static/control/multi-fpm.info but I want to be sure that it actually now runs on that version.

**Comment by Corvalan** [comment]
Update: I managed to login and checked..its still php 7.3 which is strange. \
The questions of this posts still is open but I'll open a new issue for the multi-fpm.info problem

**Comment by omega8cc** [comment]
You can verify versions used by checking files in this directory:

`/data/disk/o1/config/server_master/nginx/post.d`

You should see there files for the default and site specific includes, like

```
fpm_include_site_www.foo.com.inc
if ( $main_site_name = www.foo.com ) {
  set $user_socket "o2.70";
}
```

```
fpm_include_default.inc
set $user_socket "o2.56";
```
